contracts/contracts/BidderRegistry.sol

```diff
@@ -376,10 +373,10 @@ contract BidderRegistry is
     /**
      * @notice Sets the new fee recipient
      * @dev onlyOwner restriction
-     * @param newFeeRecipient The address to transfer the slashed funds to.
+     * @param newProtocolFeeRecipient The new address to accumulate protocol fees
      */
-    function setNewFeeRecipient(address newFeeRecipient) external onlyOwner {
-        feeRecipient = newFeeRecipient;
+    function setNewProtocolFeeRecipient(address newProtocolFeeRecipient) external onlyOwner {
+        protocolFeeTracker.recipient = newProtocolFeeRecipient;
```

Comment:
It's a good practice to generate event in case that function is called
```suggestion
        protocolFeeTracker.recipient = newProtocolFeeRecipient;
        emit ProtocolFeeRecipientUpdated(newProtocolFeeRecipient);
```